_posts/en/2021-06-08-working-in-the-open-part-2.md

```diff
@@ -0,0 +1,147 @@
+
+This means that when a small part of your work must not be shared publicly, you can exclude it and store it in a secure location instead of locking down the entire document.
+
+This way, the material that is unclassified can still be made available publicly and may be collectively worked on while the parts that are to be secured can be edited separately.
+
+By default, you should strive to keep as many of your working documents in the open section of your workspace.
+But when needed, you can save sensitive information in the private folders and keep working in the open for the rest.
+
+### Manage leaks
+
+In software development, we expect that there will be a security issue at some point, while we continue to strive to minimize that eventuality.
+
+By putting in place a remediation process, we ensure that even if there was to be a leak, appropriate mesures are in place to address it as quickly as possible.
+And we make sure that everyone knows how to run through the process, if it's not fully automated.
+
+### Open up at inception
+
+Part of working in the open is accepting that we don't know everything and that others may be able to jump in and help at any time.
+
+When one goes into draft mode, it's easy to think that we can do better before getting feedback, so the initial reaction most people have is to wait until "it's good enough".
+
+If you are worried to be slowed down by feedback on a document that is barely an outline, you probably can comfort yourself with the thought that, in practice, chances are that contributions would not come before your product or document is actually mature enough!
+
+Plus, waiting too long may put you in the position where you have not set your mind into an open mode, continuously expecting someone to read it.
+That may lead you to add a review phase before opening up to ensure that no sensitive material has been included, instead of working from the start with that mindset.
+
+### Be humble and open minded
+
+Working in the open may seem like putting yourself out there, subject to criticism and judgement of others.
+
+But remember that no one knows everything.
+Opening up is an opportunity for you and your team to have others contribute to your work and to complement what you already know if they are interested in it.
+
+We often seek other teams feedback in an structured manner and that official process can still occur while receiving live feedback in between "versions" of the documents you produce.
+
+### Manage the community
+
+An argument sometimes used for avoiding working in the open is that managing the community may end up being more work than what you get back.
+Or that receiving external contributions before a certain version is mature enough would slow you down.
+
+Based on our experience, small and continuous feedback makes the process much less cognitively intensive since contributions usually are smaller and can be more easily integrated in the workflow.
```

Comment:
In this last section I would put our responses to these two arguments (in 133 and 134) sequentially... It seems like 136 is addressing the 'slow you down' argument.  Do we actually address 133? Like is managing the community more work than what you get back?  I think no but we don't actually say that. for row 134, we do mention this above as well so not sure if we need to address both here and above?

Reply:
Aye, good point😬 Will restructure

Reply:
I've restructured and feel it's a bit better now? :thiking:

Reply:
Looks good!